### ISSUE TITLE
Add ability to set expand ration of columns

### ISSUE THREAD

**Issue by a-schild** [vaadin-component-factory/vcf-gridlayout#3]
In hte vaadin 6-8 component you could specify the expand ration of the columns this way:

```
gLayout.setColumnExpandRatio(0, 25);
gLayout.setColumnExpandRatio(1, 75);

```

**Comment by TatuLund** [comment]
There is an another version of GridLayout in Classic Components collection, which includes this feature: https://vaadin.com/docs/latest/flow/advanced/classic-components

**Comment by a-schild** [comment]
Yes, but unfortunally it requires the prime subscription for this.
I will look if I can provde a pull request to add the functionality

**Comment by a-schild** [comment]
Here #4 is a pull request which implements basic column width and expansion ratio settings

**Comment by a-schild** [comment]
Versioning should probably be 1.1.0-SNAPSHOT if you are using sematic versioning
